### PR TITLE
Add warning to remind users to use Oasis Devnet and to have DEV

### PR DESCRIPTION
**Pull request by clementfung**
Finally got the PR to work!

This is my first PR here, so please let me know if I'm not following company conventions in any way.

The dashboard warning message formatting and text are totally open to suggestion if you want something changed.

### CODE REVIEW

**Comment by willscott** [comment]
er. assuming this is the full PR. if the `[WIP]` implies there's more you're planning to add, we can re-review

File: app/index.html [hunk 1]
```diff
@@ -7,6 +7,11 @@
 </head>
 <body class="container">
 <center><h1>Oasis Secret Ballot dApp</h1></center>
+<center>
+    <div id="warning-div">
+        <p>Reminder: Please ensure you are using the Oasis Devnet (configured on MetaMask) and received DEV from the <a href="https://faucet.oasiscloud.io/">Oasis Faucet</a>.</p>
```
Comment (pro-wh) [inline app/index.html]: slight bikeshed: maybe we should link to the tutorial instead of directly to the faucet

Comment (clementfung) [reply app/index.html]: Yeah, I was slightly worried about this; happy to fix that.

**Comment by clementfung** [comment]
@pro-wh Updated this to link to the tutorial directly. Let me know if it looks okay!

File: app/index.html [hunk 2]
```diff
@@ -7,6 +7,12 @@
 </head>
 <body class="container">
 <center><h1>Oasis Secret Ballot dApp</h1></center>
+<center>
+    <div id="warning-div">
+        <p>Reminder: Please ensure you are using the Oasis Devnet (configured on MetaMask) and received sufficient DEV from 
+            the Oasis Faucet. <br><a href="https://docs.oasiscloud.io/en/latest/secret-ballot/">See tutorial for details</a>.</p>
```
Comment (pro-wh) [inline app/index.html]:
```suggestion
            the Oasis faucet. <br><a href="https://docs.oasiscloud.io/en/latest/secret-ballot/">See tutorial for details</a>.</p>
```

I think we haven't been capitalizing "faucet" in our other copy

**Pull request merged**